tests/test_datacube.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+import sys
+sys.path.append('../scripts/')
+from datacube import *
+
+def test_read_data(data_path):
+    ds = read_data(data_path)
+
+    for var in ['dv', 'rlat', 'rlon', 'lat', 'lon']: 
+        assert var in ds.variables, 'Key \'{}\' not properly inserted into Dataset'.format(var)
+
+    assert len(ds['dv'].shape) == 3, 'Incorrect dimension size after loading'
```

Comment:
This single function is actually several tests, and could be improved as follows:
* [Parameterize](https://doc.pytest.org/en/latest/parametrize.html) a test over the variable names (`var` in your code).
* Possibly create two different tests, one for existence and one for shape, or, more likely, add the shape as part of the parametrization of the (single test).

We can talk about this if you want.

Reply:
Heh, I missed your comment, but I too came here to sing the praises of `parametrize`. :)